target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -18,12 +18,19 @@ ath10kcal_extract() {
 	local part=$1
 	local offset=$2
 	local count=$3
-	local mtd
+	local mtd size1 size2
 
 	mtd=$(find_mtd_chardev $part)
 	[ -n "$mtd" ] || \
 		ath10kcal_die "no mtd device found for partition $part"
 
+	# Check that the calibration data size in header equals the desired size
```

Comment:
I'm worried about possible regression if we apply this check globally, for all boards.
Are you able point out any information about header layout and if it's the same for all ath10k chipsets?

Reply:
Yes, the header layout is the same for all current ath10k chips, both cal and pre-cal data. The same check is made in Qualcomm driver.

Reply:
Can you tell me where I can find it (link or what source file look into)?

Reply:
For example https://git.kos.org.cn/openkos/qca-wifi/blob/master/qca-wifi/qca-wifi-10.2.4.58.1/offload/os/linux/tools/athdiag/qc98xx_eeprom.h look for base_eep_header.

Reply:
Great, thanks!
I think it's safe to keep that check for all boards.